glastopf/glastopf.cfg.dist

```diff
@@ -99,3 +99,6 @@ banner = Apache/2.0.48
 google_meta =
 #http://www.bing.com/toolbox/webmaster
 bing_meta =
+
+[sensor]
+sensorid = hooli
```

Comment:
hm, how about setting this to None and change it to a UUID if not set on the first startup?